AzureMonitorAgent/agent.py

```diff
@@ -1181,8 +1181,8 @@ def metrics_watcher(hutil_error, hutil_log):
                 f.close()
             
             if fluent_port != '' and os.path.isfile(FluentCfgPath):
-                portSetting = "    Port    "  + fluent_port
-                defaultPortSetting = 'Port    '
+                portSetting = "    Port                       "  + fluent_port
```

Comment:
Is there a reason we can't do a regex match here? Needing to match on the precise amount of space feels fragile.